src/test/java/com/fasterxml/jackson/databind/deser/CustomDeserializersTest.java

```diff
@@ -20,9 +19,8 @@
 import com.fasterxml.jackson.databind.util.NameTransformer;
 import com.fasterxml.jackson.databind.util.StdConverter;
 
-import static org.junit.jupiter.api.Assertions.*;
-
 import static com.fasterxml.jackson.databind.testutil.DatabindTestUtil.*;
+import static org.junit.jupiter.api.Assertions.*;
```

Comment:
Here also. Same ordering rule applies to [static import](https://arc.net/l/quote/cpsxuvhk).
May I ask which tool you used to refactor this?
May I can help configuring also.

Reply:
Oh, I had overlooked that part, now static imports are just sorted lexically without empty lines.
I am using OpenRewrite and have written a custom recipe for that (or rather defined a style for the existing order imports recipe). Here is the code: https://github.com/timo-a/rewrite-recipe-starter/blob/main/src/main/java/io/github/timoa/misc/JacksonImportStyle.java. This PR https://github.com/timo-a/rewrite-recipe-starter/pull/5 should fix the order. Could you have a look at the added test and let me know if the proposed ordering is correct?

You can try it out locally by adding the following to the pom of your project and running `mvn -P openrewrite org.openrewrite.maven:rewrite-maven-plugin:run`
```
    <profile>
      <id>openrewrite</id>
      <!-- `mvn -P openrewrite org.openrewrite.maven:rewrite-maven-plugin:run` -->
      <build>
        <plugins>
          <plugin>
            <groupId>org.openrewrite.maven</groupId>
            <artifactId>rewrite-maven-plugin</artifactId>
            <version>5.41.0</version>
            <configuration>
              <activeRecipes>
                <recipe>org.openrewrite.java.OrderImports</recipe>
              </activeRecipes>
              <activeStyles>
                <style>io.github.timoa.misc.JacksonImportStyle</style>
              </activeStyles>
              <failOnDryRunResults>true</failOnDryRunResults>
            </configuration>
            <dependencies>
              <dependency>
                <groupId>io.github.timo-a</groupId>
                <artifactId>rewrite-recipe-starter</artifactId>
                <version>0.5.0-SNAPSHOT</version>
              </dependency>
            </dependencies>
          </plugin>
        </plugins>
      </build>
    </profile>
```
You'll probably have to adapt the version of the dependency. 

- If you want to use the latest published version (which does not yet sort correctly) then set it to `0.4.1`.
- If you want to use an unpublished version, like the one from my PR then run  (gradle) `publishToMavenLocal` and inspect your .m2 for the correct version.

Reply:
Cool stuff, you think we can integrate this into maven build system, @timo-a ?
Expectation is import ordering is automated.
But without giving too much overhead  -- saying this incase OpenRewrite tries to full scan everything everytime or something, just gut-feeling tho.

Reply:
If I read PR correctly, it would only do this when PR is updated, as part of CI (GH Action triggers).
Which wouldn't be too mad. So it would not change regular Maven build process, I think.

Reply:
Nice, another improvement idea!

Reply:
Indeed!